Formula/make.rb

```diff
@@ -28,6 +28,27 @@ def install
 
     system "./configure", *args
     system "make", "install"
+
+    if build.without? "default-names"
+      (libexec/"gnubin").install_symlink bin/"gmake" =>"make"
+      (libexec/"gnuman/man1").install_symlink man1/"gmake.1" => "make.1"
+    end
+  end
+
+  def caveats
+    if build.without? "default-names" then <<-EOS.undent
```

Comment:
Done in 2f84eb40da8d6113a9271a40c137448182f5fa50
A shameless copy from `findutils` =D